Can't write LDAP configuration in 3.7 (or newer)

**Describe the bug**
It is not possible to write new users to LDAP

**To Reproduce**
Steps to reproduce the behavior:
1. Go to the "Users" page
2. Click on "New"
3. Click on "New user"
4. Filling out the form with valid data
5. Leaving "LDAP group" default to "No selection"
6. Click on "Save" and create new user
7. Edit the new user you have just created
8. Select the specific LDAP group (for example Local LDAP)
9. Click on "Write LDAP configuration"
10. See error "NameNotFoundException" in red box at top

**Expected behavior**
That the selected user is written to LDAP and a green success message is displayed at the top

**Screenshots**
![ldap_error1](https://github.com/user-attachments/assets/412d00a0-5860-43dd-b6d9-0187ba4924a1)

**Release**
Inital noticed in version 3.7.0. but also tested with current master branch (3.8.0-snapshot)

**Desktop (please complete the following information):**
 - OS: Kubuntu 22.04
 - Browser Firefox
 - Version 129.0.1

**Additional context**
The same procedure for writing the user to the LDAP works without problems in version 3.6.3
Tested with empty / fresh installed systems

**Logs**
```
[ERROR] 2024-08-30 14:12:53.007 [http-nio-8080-exec-9] UserForm - Could not generate ldap entry
javax.naming.NameNotFoundException: null
        at org.kitodo.production.ldap.LdapUser.getAttributes(LdapUser.java:283) ~[classes/:3.8.0-SNAPSHOT]
        at com.sun.jndi.ldap.Obj.determineBindAttrs(Obj.java:573) ~[?:?]
        at com.sun.jndi.ldap.LdapCtx.c_bind(LdapCtx.java:440) ~[?:?]
        at com.sun.jndi.ldap.LdapCtx.c_bind(LdapCtx.java:409) ~[?:?]
        at com.sun.jndi.toolkit.ctx.ComponentContext.p_bind(ComponentContext.java:612) ~[?:?]
        at com.sun.jndi.toolkit.ctx.PartialCompositeContext.bind(PartialCompositeContext.java:201) ~[?:?]
        at com.sun.jndi.toolkit.ctx.PartialCompositeContext.bind(PartialCompositeContext.java:191) ~[?:?]
        at javax.naming.InitialContext.bind(InitialContext.java:417) ~[?:?]
        at org.kitodo.production.services.data.LdapServerService.createNewUser(LdapServerService.java:198) ~[classes/:3.8.0-SNAPSHOT]
        at org.kitodo.production.forms.UserForm.writeUserAtLdapServer(UserForm.java:501) ~[classes/:3.8.0-SNAPSHOT]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
        at org.apache.el.parser.AstValue.invoke(AstValue.java:252) ~[tomcat9-jasper-el-9.0.58.jar:9.0.58]
        at org.apache.el.MethodExpressionImpl.invoke(MethodExpressionImpl.java:266) ~[tomcat9-jasper-el-9.0.58.jar:9.0.58]
        at org.jboss.weld.util.el.ForwardingMethodExpression.invoke(ForwardingMethodExpression.java:40) ~[weld-servlet-2.4.8.Final.jar:2.4.8.Final]
        at org.jboss.weld.el.WeldMethodExpression.invoke(WeldMethodExpression.java:50) ~[weld-servlet-2.4.8.Final.jar:2.4.8.Final]
        at org.jboss.weld.util.el.ForwardingMethodExpression.invoke(ForwardingMethodExpression.java:40) ~[weld-servlet-2.4.8.Final.jar:2.4.8.Final]
        at org.jboss.weld.el.WeldMethodExpression.invoke(WeldMethodExpression.java:50) ~[weld-servlet-2.4.8.Final.jar:2.4.8.Final]
        at org.apache.myfaces.view.facelets.el.ContextAwareTagMethodExpression.invoke(ContextAwareTagMethodExpression.java:96) ~[myfaces-impl-2.3.10.jar:2.3.10]
        at org.apache.myfaces.application.ActionListenerImpl.processAction(ActionListenerImpl.java:74) ~[myfaces-impl-2.3.10.jar:2.3.10]
        at org.springframework.faces.webflow.FlowActionListener.processAction(FlowActionListener.java:71) ~[spring-faces-2.5.1.RELEASE.jar:2.5.1.RELEASE]
        at org.springframework.faces.model.SelectionTrackingActionListener.processAction(SelectionTrackingActionListener.java:64) ~[spring-faces-2.5.1.RELEASE.jar:2.5.1.RELEASE]
        at javax.faces.component.UICommand.broadcast(UICommand.java:120) ~[myfaces-api-2.3.10.jar:2.3.10]
        at javax.faces.component.UIViewRoot._broadcastAll(UIViewRoot.java:1255) ~[myfaces-api-2.3.10.jar:2.3.10]
        at javax.faces.component.UIViewRoot.broadcastEvents(UIViewRoot.java:420) ~[myfaces-api-2.3.10.jar:2.3.10]
        at javax.faces.component.UIViewRoot._process(UIViewRoot.java:1741) ~[myfaces-api-2.3.10.jar:2.3.10]
        at javax.faces.component.UIViewRoot.processDecodes(UIViewRoot.java:942) ~[myfaces-api-2.3.10.jar:2.3.10]
        at org.apache.myfaces.lifecycle.ApplyRequestValuesExecutor.execute(ApplyRequestValuesExecutor.java:42) ~[myfaces-impl-2.3.10.jar:2.3.10]
        at org.apache.myfaces.lifecycle.LifecycleImpl.executePhase(LifecycleImpl.java:195) ~[myfaces-impl-2.3.10.jar:2.3.10]
        at org.apache.myfaces.lifecycle.LifecycleImpl.execute(LifecycleImpl.java:142) ~[myfaces-impl-2.3.10.jar:2.3.10]
        at javax.faces.webapp.FacesServlet.service(FacesServlet.java:204) ~[myfaces-api-2.3.10.jar:2.3.10]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53) ~[tomcat9-websocket-9.0.58.jar:9.0.58]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.kitodo.production.servletfilter.EncodingFilter.doFilter(EncodingFilter.java:68) ~[classes/:3.8.0-SNAPSHOT]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.omnifaces.filter.GzipResponseFilter.doFilter(GzipResponseFilter.java:183) ~[omnifaces-3.13.3.jar:3.13.3]
        at org.omnifaces.filter.HttpFilter.doFilter(HttpFilter.java:108) ~[omnifaces-3.13.3.jar:3.13.3]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:337) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.kitodo.production.security.SecurityObjectAccessFilter.doFilter(SecurityObjectAccessFilter.java:89) ~[classes/:3.8.0-SNAPSHOT]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.java:115) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.java:81) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:122) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:116) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:126) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:81) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:109) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:149) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.session.ConcurrentSessionFilter.doFilter(ConcurrentSessionFilter.java:147) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.session.ConcurrentSessionFilter.doFilter(ConcurrentSessionFilter.java:125) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:223) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:217) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:103) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:89) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:90) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:75) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.29.jar:5.3.29]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:112) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:82) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:55) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.29.jar:5.3.29]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.session.DisableEncodeUrlFilter.doFilterInternal(DisableEncodeUrlFilter.java:42) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.29.jar:5.3.29]
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:346) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:221) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:186) ~[spring-security-web-5.7.11.jar:5.7.11]
        at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:354) ~[spring-web-5.3.29.jar:5.3.29]
        at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:267) ~[spring-web-5.3.29.jar:5.3.29]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.logging.log4j.web.Log4jServletFilter.doFilter(Log4jServletFilter.java:71) ~[log4j-web-2.19.0.jar:2.19.0]
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:540) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:359) ~[tomcat9-catalina-9.0.58.jar:9.0.58]
        at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399) ~[tomcat9-coyote-9.0.58.jar:9.0.58]
        at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) ~[tomcat9-coyote-9.0.58.jar:9.0.58]
        at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889) ~[tomcat9-coyote-9.0.58.jar:9.0.58]
        at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1735) ~[tomcat9-coyote-9.0.58.jar:9.0.58]
        at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) ~[tomcat9-coyote-9.0.58.jar:9.0.58]
        at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) ~[tomcat9-util-9.0.58.jar:9.0.58]
        at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) ~[tomcat9-util-9.0.58.jar:9.0.58]
        at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) ~[tomcat9-util-9.0.58.jar:9.0.58]
        at java.lang.Thread.run(Thread.java:829) ~[?:?]
```

--- a/Kitodo/src/main/java/org/kitodo/production/ldap/LdapUser.java
+++ b/Kitodo/src/main/java/org/kitodo/production/ldap/LdapUser.java
@@ -279,7 +279,7 @@ public class LdapUser implements DirContext {
 
     @Override
     public Attributes getAttributes(String name) throws NamingException {
-        if (StringUtils.isBlank(name)) {
+        if (!StringUtils.isBlank(name)) {
             throw new NameNotFoundException();
         }
         return (Attributes) this.attributes.clone();
@@ -292,7 +292,7 @@ public class LdapUser implements DirContext {
 
     @Override
     public Attributes getAttributes(String name, String[] ids) throws NamingException {
-        if (StringUtils.isBlank(name)) {
+        if (!StringUtils.isBlank(name)) {
             throw new NameNotFoundException();
         }
 

--- a/Kitodo/src/main/java/org/kitodo/production/ldap/LdapUser.java
+++ b/Kitodo/src/main/java/org/kitodo/production/ldap/LdapUser.java
@@ -279,7 +279,7 @@ public class LdapUser implements DirContext {
 
     @Override
     public Attributes getAttributes(String name) throws NamingException {
-        if (!StringUtils.isBlank(name)) {
+        if (StringUtils.isNotBlank(name)) {
             throw new NameNotFoundException();
         }
         return (Attributes) this.attributes.clone();
@@ -292,7 +292,7 @@ public class LdapUser implements DirContext {
 
     @Override
     public Attributes getAttributes(String name, String[] ids) throws NamingException {
-        if (!StringUtils.isBlank(name)) {
+        if (StringUtils.isNotBlank(name)) {
             throw new NameNotFoundException();
         }
 

--- a/Kitodo/src/main/resources/messages/messages_en.properties
+++ b/Kitodo/src/main/resources/messages/messages_en.properties
@@ -680,7 +680,7 @@ lastPage=Last page
 lastUpdatedBy=Last updated by
 later=Please try again in a few minutes.
 ldapKonfigurationSchreiben=Write LDAP configuration
-ldapWritten=LDAP configuration successfully written for user\: ????
+ldapWritten=LDAP configuration successfully written for user\:
 # ldapgroupSaving is used in ldapgroupEdit.xhtml - line 39
 ldapGroupSaving=Saving LDAP group...
 ldapGroup=LDAP group

